src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -159,4 +160,105 @@ public void nullUserEntityTitle(){
       Assert.assertEquals(expected, actual);
   }
 
+    // Given an empty string for game id, addGameToUser should return false
+  @Test
+  public void emptyGameIdFails() {
+
+    Entity userEntity = new Entity("User");
```

Comment:
Also I think this line will be declared as "user" with lower case "u"

Reply:
yep, done!